setup/sql/dbupdate_04.php

```diff
@@ -23909,7 +23909,7 @@ function writeCtrlClassEntry(ilPluginSlot $slot, array $plugin_data) {
 				$timestamp = $certificateFile->getMTime();
 
 				if (false !== $content) {
-					$backgroundImagePath = $relativePath . $relativePath . '/background.jpg';
```

Comment:
This should be already fixed: https://github.com/ILIAS-eLearning/ILIAS/commit/6b52a2db68228c1771d69bbd4463ed8002217e8b